Adding info for conditions for autodiscovery (#921)
elastic/ingest-docs: docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

[[conditions-based-autodiscover]]
= Conditions based autodiscover

You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. Refer to <<elastic-agent-input-configuration>> to get an idea.

> IMPORTANT: Condition definintion is supported both in *{agent} by {fleet}* and in *standalone* scenarios.

Reviewer: ```suggestion
IMPORTANT: Condition definintion is supported both in *{agent} managed by {fleet}* and in *standalone* scenarios.
```

> IMPORTANT: Condition definintion is supported both in *{agent} managed by {fleet}* and in *standalone* scenarios.

Reviewer 2: ```suggestion
IMPORTANT: Condition definition is supported both in *{agent} managed by {fleet}* and in *standalone* scenarios.
```

> Read more information for variables and conditions in input configurations in <<dynamic-input-configuration>>
> You can find available variables of autodiscovery in <<kubernetes-provider>>  

Reviewer: ```suggestion
For more information about variables and conditions in input configurations, refer to <<dynamic-input-configuration>>.
You can find available variables of autodiscovery in <<kubernetes-provider>>.
```
Just rephrasing a little bit to match what we have in some other docs.
